azext_edge/edge/commands_edge.py

```diff
@@ -124,6 +124,7 @@ def init(
     mq_authn_name: str = "authn",
     mq_broker_config_file: Optional[str] = None,
     mq_insecure: Optional[bool] = None,
+    dataflow_profile_instances: int = 1,
```

Comment:
can we not get away with no dataflow profile instances?

Reply:
What is the concern with exposing this ?

Reply:
oh I meant as in make an aio deployment without the dataflow instance - sorry, I took a motion sickness pill and am pretty sleepy now

Reply:
You can deploy AIO without a dataflow profile, but every dataflow requires a profile. And dataflows are common scenarios in AIO. Also, DOE needs the default profile available.